os/pm/pm_changestate.c

```diff
@@ -242,6 +242,9 @@ int pm_changestate(enum pm_state_e newstate)
 		pm_changeall(newstate);
 #ifdef CONFIG_PM_METRICS
 		pm_metrics_update_changestate();
+#endif
+#ifdef CONFIG_LCD
```

Comment:
Do we need separate config for PM LCD like (CONFIG_PM_LCD) or LCD feature config(CONFIG_LCD) is sufficient here?

Reply:
I think its sufficient